### PR TITLE
[GooglePlayPromote] Migrated to Node10

### PR DESCRIPTION
**Pull request by sdobrodeev**
**Task name**: google-play-promote

**Description**: Migrated task to Node 10.

**Documentation changes required:** No

**Added unit tests:** No

**Attached related issue:** No

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by anatolybolshakov — APPROVED** [review]

**Comment by anatolybolshakov** [comment]
Let's make buddy testing before we merge it

File: Tasks/google-play-promote/task.loc.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   ],
   "version": {
     "Major": "3",
-    "Minor": "174",
+    "Minor": "182",
```
Comment (anatolybolshakov) [inline Tasks/google-play-promote/task.loc.json:17]: Please bump version to current sprint number

Comment (egor-bryzgalov) [reply Tasks/google-play-promote/task.loc.json]: Thanks! Fixed

**Comment by egor-bryzgalov** [comment]
Tested manually.

**Comment by sdobrodeev** [comment]
Tested manually. No issues found.

**Review by DaniilShmelev — APPROVED** [review]

**Pull request merged**